From d767e709fd686153b863aa67ae078818508b4d34 Mon Sep 17 00:00:00 2001
From: Li Jun <B47624@freescale.com>
Date: Thu, 16 Oct 2014 22:18:02 +0800
Subject: [PATCH 0143/5242] MLK-9618-9 usb: chipidea: otg: change TA_BIDL_ADIS
 to be 160ms

commit  799eca59187bbb48715b64594ef618325b0b7e6a from
https://source.codeaurora.org/external/imx/linux-imx.git

This patch is to fix one OTG HNP timing requirement, TA_BIDL_ADIS should be
in range of 155ms~200ms per OTG spec 2.0 reverion 1.1a.

Signed-off-by: Li Jun <b47624@freescale.com>
(cherry picked from commit a6de1be200984b01d784f6558229adbb67fcb380)
(cherry picked from commit a94422371328c620f4b37520868ad3f96ec2e165)
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 drivers/usb/chipidea/otg_fsm.h |    5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/usb/chipidea/otg_fsm.h b/drivers/usb/chipidea/otg_fsm.h
index 5d9c0e6..756f423 100644
--- a/drivers/usb/chipidea/otg_fsm.h
+++ b/drivers/usb/chipidea/otg_fsm.h
@@ -35,9 +35,8 @@
 					 * TA_AIDL_BDIS: section 5.5, Table 5-1
 					 */
 /* B-Idle to A-Disconnect */
-#define TA_BIDL_ADIS         (500)	/* TA_BIDL_ADIS: section 5.2.1
-					 * 500ms is used for B switch to host
-					 * for safe
+#define TA_BIDL_ADIS         (160)	/* TA_BIDL_ADIS: section 5.2.1
+					 * 155ms ~ 200 ms
 					 */
 
 #define TA_DP_END             (200)
-- 
1.7.9.5

